arviz/tests/base_tests/test_plots_bokeh.py

```diff
@@ -1037,8 +1037,9 @@ def test_plot_bpv(models, kwargs):
 
 
 def test_plot_bpv_discrete():
-    fake = {"a": np.random.poisson(2.5, 1000)}
-    fake_model = from_dict(posterior_predictive=fake, observed_data=fake)
+    fake_obs = {"a": np.random.poisson(2.5, 100)}
+    fake_pp = {"a": np.random.poisson(2.5, (10, 100))}
```

Comment:
As a 2 dim array this will be understood as a `(chain, draw)` array, you should add a dimension at the beggining.